### DISCUSSION TITLE
Keyword List - Indexed Report

### DISCUSSION THREAD

**Discussion by gisms** [sepinf-inc/IPED#373] [Q&A]
Hi!!!

What is the purpose of importing a keyword list when generating an indexed report?
I saw no difference when using this option.

Did the proposal involve being able to export the report without the inclusion of a tag, but only the list of words?

Thank you.

**Comment by hauck-jvsh** [comment]
When you import a keyword list, all the keyword appears in the search field. I use this feature when the annalists must search for all of those terms in a lot of materials, making this task easier.

**Comment by lfcnassif** [comment]
Moved to discussions.

It can also be used with "blind" profile (automatic extraction cases), so investigators do not have to type each relevant keyword to search for it, like @hauck-jvsh said.

**Reply by gisms** [reply]
Now I understand, it serves to generate the digital report with the words in the search bar.
I thought that somehow they would also be in the htm report.
We were looking for a way to generate the htm report with just the keywords, without the need to create the bookmarks. I thought this functionality had something to do with it.

thanks guys.

**Reply by lfcnassif** [reply]
You can create automatic bookmarks at the end of the processing using javascript, e.g. using the results of some searches, take a look at https://github.com/sepinf-inc/IPED/blob/master/iped-app/resources/config/profiles/en/default/conf/scripts/ExampleScriptTask.js

**Reply by gisms** [reply]
Thank u for this idea, i will look at it.
